andela/bad_ass-ah-frontend
# #164489898 view notifications

**What does this PR do?**

- This PR adds receiving notifications functionality

**Description of Task to be completed?**

- If a user has favorited an article, he can receive a notification whenever there is a change on that article like a new comment on the article, article edited, etc.

**How should this be manually tested?**

- Go to the app and log in
- If you have notifications you will see a bell icon with a red badge in the navbar
- When you click on one of them it is removed(It is marked as read)

**What are the relevant pivotal tracker stories?**

[#164489898](https://www.pivotaltracker.com/story/show/164489898)

**Screenshots (if appropriate)**

<img width="1425" alt="Screen Shot 2019-06-14 at 09 27 20" src="https://user-images.githubusercontent.com/32579116/59493063-ad8f6e00-8e8a-11e9-83b1-5a26f0a88bfd.png">
